From: David Paleino <dapal@debian.org>
Subject: use python-iniparse instead of ConfigParser,
 because the former leaves the config files sorted, so that no spurious
 changes appear when /etc/ is e.g. stored with etckeeper.
Forwarded: no
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=568326

---
 wicd/configmanager.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- wicd.orig/wicd/configmanager.py
+++ wicd/wicd/configmanager.py
@@ -26,7 +26,7 @@ reusable for other purposes as well.
 
 import os, copy
 
-from ConfigParser import RawConfigParser, ParsingError
+from iniparse.compat import RawConfigParser, ParsingError
 
 from wicd.misc import Noneify, to_unicode
 
